packages/example-playground/src/examples/reactive/readme.md

```diff
@@ -0,0 +1,72 @@
+# Reactive views
+
+## Leading principals
+
+- From a developer perspective, all the code that can be affected by a change to state/store/prop runs synchronously and sequentially, meaning when change tree branches, should execute as it would if not reactive
+- Only affected code should run as a reaction to change
+- It should be clear to the developer which parts of the code are reactive and which aren't
+
+### types of variables
+
+#### Persistent vars
+
+- props defined by the parent component or via component API
```

Comment:
maybe add readonly?

alternatively we can support two way binding with some convention